ios/TransactionListViewCell.swift

```diff
@@ -76,7 +76,7 @@ class TransactionListViewCell: TransactionListBaseCell {
         let url = URL(string: "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/\(transaction.address!)/logo.png");
         coinImage.sd_setImage(with: url) { (image, error, cache, urls) in
           if (error != nil) {
-            let colorIndex = transaction.symbol.lowercased().utf8.compactMap{ Int($0) }.reduce(0, +) % TransactionListViewCell.avatarColors.count
+            let colorIndex = transaction.address!.lowercased().utf8.compactMap{ Int($0) }.reduce(0, +) % TransactionListViewCell.avatarColors.count
```

Comment:
@brunobar79 @osdnk can one of you let me know if this `!` was necessary? xcode complained and suggested i add either a `!` or an `?` but unsurrreeeeee

Reply:
I think with `address!` it'll crash if `address` doesn't exist

Reply:
at the top of that block is `if transaction.symbol != nil {`

which might be why it wasn't complaining about `symbol`

Reply:
what im confuse about is it seems like all this logic is already wrapped by `if (address != nil)` so like how could `address` not exist? https://github.com/rainbow-me/rainbow/blob/952683f2bb80fb70b3bb36da05c7ba31d9ef1556/ios/TransactionListViewCell.swift#L75

Reply:
missed the `if address != nil`

you should be good to force unwrap with `!` then because `address` will always exist